2_process/src/process_data.R

```diff
@@ -2,21 +2,44 @@
 
 
 load_data <- function(mendota_file) {
-  read_csv(mendota_file, col_types = 'iccd') %>%
-    filter(str_detect(exper_id, 'similar_[0-9]+')) %>%
-    mutate(col = case_when(
-      model_type == 'pb' ~ '#1b9e77',
-      model_type == 'dl' ~'#d95f02',
-      model_type == 'pgdl' ~ '#7570b3'
-    ), pch = case_when(
-      model_type == 'pb' ~ 21,
-      model_type == 'dl' ~ 22,
-      model_type == 'pgdl' ~ 23
-    ), n_prof = as.numeric(str_extract(exper_id, '[0-9]+')))
-  
+                      read_csv(mendota_file, col_types = 'iccd') %>%
```

Comment:
again huge indentation? I prefer (and I think it's pretty standard) 2 spaces per indentation level (e.g., `read_csv` would be 2 spaces in, `filter` and `mutate` would be 4 spaces in)